OpenOversight/app/models/database.py

```diff
@@ -101,18 +105,23 @@ def total_documented_assignments(self):
             .count()
         )
 
-    @cached(cache=DB_CACHE, key=db_model_cache_key(KEY_DEPT_TOTAL_INCIDENTS))
+    @cached(cache=DB_CACHE, key=model_cache_key(KEY_DEPT_TOTAL_INCIDENTS))
     def total_documented_incidents(self):
         return (
             db.session.query(Incident).filter(Incident.department_id == self.id).count()
         )
 
-    @cached(cache=DB_CACHE, key=db_model_cache_key(KEY_DEPT_TOTAL_OFFICERS))
+    @cached(cache=DB_CACHE, key=model_cache_key(KEY_DEPT_TOTAL_OFFICERS))
     def total_documented_officers(self):
         return (
             db.session.query(Officer).filter(Officer.department_id == self.id).count()
         )
 
+    @staticmethod
+    def remove_database_cache_entry(dept_id: int, update_type: [str]) -> None:
+        """Remove the Department model key from the cache if it exists."""
+        remove_database_cache_entry(Department(id=dept_id), update_type)
```

Comment:
I made this function so I didn't have to type out `Department(id=dept_id)` every time I tried to remove something from the cache.